gui/slick/js/core.js

```diff
@@ -298,6 +298,20 @@ var SICKRAGE = {
                     }
                 });
             });
+
+            $('#branchForceUpdate').on('click', function() {
+                $(this).attr('disabled', 'disabled');
+                $('#git_reset_branches').empty();
+                $.getJSON(srRoot + '/home/branchForceUpdate', function(data) {
+                    data.resetBranches.forEach(function(branch) {
+                        $('#git_reset_branches').append('<option value="' + branch + '" selected="selected" >' + branch + '</option>');
+                    });
+                    data.branches.forEach(function(branch) {
+                        $('#git_reset_branches').append('<option value="' + branch + '" >' + branch + '</option>');
+                    });
+                    $(this).prop('disabled', false);
+                });
+            });
```

Comment:
@OmgImAlexis I manage to make JS to put resetBranches on top and not reset branches in the botton